integration-tests/src/test/java/com/datastax/oss/driver/mapper/NullSavingStrategyIT.java

```diff
@@ -171,6 +184,10 @@ ProductSimpleDaoGlobalLevelDoNotSetOverrideSetToNull productDaoLocalOverride(
     ProductSimple findById(UUID productId);
   }
 
+  @Dao
+  public interface ProductSimpleDaoSetToNullFromParentInterface
+      extends ProductSimpleDaoImplicit, SetToNull {}
```

Comment:
Why do we need to extend `ProductSimpleDaoImplicit` (instead of inlining it in the `ProductSimpleDaoSetToNullFromParentInterface`)?
It seems that we are testing two aspects - maybe it will be better to split it into two tests?
Or maybe we should also test a case where ProductSimpleDaoImplicit should extend `SetToNull`?

Reply:
I chose to use `ProductSimpleDaoImplicit` because it provides methods we can use and isn't already annotated with `DefaultNullSavingStrategy`.  So this is to verify that a parent annotation is inherited.  We could have redefined the `ProductSimpleDaoImplicit`s methods here, but this seemed easier.